text/projects/Krzysztof_Michalik/Wee_LCP.tex

```diff
@@ -0,0 +1,179 @@
+\documentclass{article}
+\usepackage[utf8]{inputenc}
+
+\title{Wee LCP description}
+\author{krzysztof.michalik }
+\date{May 2022}
+
+\usepackage[T1]{fontenc}
+\usepackage[polish]{babel}
+\usepackage{amsfonts}
+\usepackage{amsmath}
+
+\usepackage{graphicx}
+\usepackage{tikz}
+\usetikzlibrary{positioning}
+\usepackage{geometry}
+
+\begin{document}
+
+\maketitle
+
+\section{Wstęp}
+
+Praca Johannesa Fishera zatytułowana Wee LCP przedstawia sposób reprezentowania struktury LCP w pamięci wykorzystującą $o(n) = O(\frac{n}{\log{\log{n}}})$ bitów, gdzie $n$ jest długością początkowego słowa $T$ dla którego obliczamy LCP.
+
+Autor zakłada dodatkowo, że:
+\begin{enumerate}
+    \item Dostępna jest tablica sufiksowa dla słowa $T$ oraz samo to słowo.
+    \item Dostęp do $i$-tej litery słowa $T$ zajmuje czas $O(1)$.
+    \item Dostęp do tablicy sufiksowej jest w czasie $t_A = \Omega(\log{n}^{\epsilon})$, gdzie $\epsilon$ jest pewną stałą.
```

Comment:
$\log{n}^{\epsilon}$ is ambigious - please change it to $\log^{\epsilon}{n}$ (and elsewhere too)